add LimitRange section, make small improvements to the VPA write-up (#53395)
kubernetes/website: content/en/docs/concepts/workloads/autoscaling/vertical-pod-autoscale.md

The Recommender analyzes both current and historical resource usage data (CPU and memory) for each Pod targeted by the VerticalPodAutoscaler. It examines:
- Historical consumption patterns over time to identify trends

> - Peak usage and variance to ensure sufficient headroom

Reviewer: Good catch!

In the _Off_ update mode, the VPA recommender still analyzes resource usage and generates
recommendations, but these recommendations are not automatically applied to Pods.

> The recommendations are only stored in the VPA object's `.status` field. In this mode, only the recommender is active.

Reviewer: I’d prefer not to change this. All components are active -  they just aren’t performing any actions.

Author: You're right all the VPA components are active, I removed those lines in https://github.com/kubernetes/website/pull/53395/commits/fd7fd6237621c8d9c7700f29ffd3add07da428a1

### Initial {#updateMode-Initial}

> In _Initial_ mode, VPA only sets resource requests when Pods are first created. It does not update resources for already running Pods, even if recommendations change over time. The recommendations apply when you modify mutable fields in the workload API, such as a Deployment, which triggers Pod recreation, or when you delete the Pods manually.

Reviewer: ```suggestion
In _Initial_ mode, VPA only sets resource requests when Pods are first created. It does not update resources for already running Pods, even if recommendations change over time. The recommendations apply only during Pod creation.
```

The `controlledValues` field determines whether VPA controls resource requests, limits, or both:

RequestsAndLimits
: VPA sets both requests and limits. The limit scales proportionally to the request based on the request-to-limit ratio defined in the Pod spec. This is the default mode.

RequestsOnly
: VPA only sets requests, leaving limits unchanged. Limits are respected and can still trigger throttling or out-of-memory kills if usage exceeds them.

See [requests and limits](/docs/concepts/configuration/manage-resources-containers/#requests-and-limits) to learn more about those two concepts.

> ## LimitRange resouces

Reviewer 2: Typo

```suggestion
## LimitRange resources
```
